components/cluster/command/clean.go

```diff
@@ -33,13 +34,24 @@ func newCleanCmd() *cobra.Command {
 			clusterName := args[0]
 			teleCommand = append(teleCommand, scrubClusterName(clusterName))
 
-			return manager.CleanCluster(clusterName, gOpt, cleanOpt, skipConfirm, withLog)
+			if cleanALl {
+				cleanOpt.CleanupData = true
+				cleanOpt.CleanupLog = true
+			}
+
+			if !(cleanOpt.CleanupData || cleanOpt.CleanupLog) {
+				return perrs.Errorf("at least one of `--all` `--data` `--log` should be specified")
```

Comment:
Use a more concrete suggestion instead of an error. e.g:
```
Please choose one of the following commands to clean the cluster:

    tiup cluster clean %s --data                  # Only clean data in the cluster
    tiup cluster clean %s --log                   # Only clean log in the cluster
    tiup cluster clean %s --all                   # Both data and log will be cleaned.
```